tests/test_one_step_setup.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+"""Test that the first time installation flow described by
+https://www.pantsbuild.org/install.html#recommended-installation works as expected."""
+
+import os
+import subprocess
+from pathlib import Path
+
+
+def test_runs_on_clean_directory(tmp_path: Path) -> None:
+
+    cwd = os.getcwd()
+
+    proc = subprocess.run(
+        ["/bin/bash", os.path.join(cwd, "one_step_setup.sh")],
+        cwd=tmp_path,
+    )
+
+    assert proc.returncode == 0
```

Comment:
maybe worth also asserting that stdout has something we want? deleting all the contents of this script would still cause this test to pause.

Reply:
Good idea. I've added that to the tests.